manifests/tasks/object-name-sanitise.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: object-name-sanitise
+spec:
+  description: This Task can be used to sanitise a string that will be used for naming a k8s object. This task converts the string to a value that can be used for an object. 
```

Comment:
Maybe it would be good to include how it is sanitized (forbidden chars replaced with `-`)?